app/build.gradle

```diff
@@ -28,4 +28,6 @@ dependencies {
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
     implementation 'com.google.android.material:material:1.0.0-rc01'
     api 'com.theartofdev.edmodo:android-image-cropper:2.8.+'
+    implementation 'androidx.recyclerview:recyclerview:1.1.0-beta01'
+    implementation 'androidx.cardview:cardview:1.0.0'
```

Comment:
nit: should be grouped together. implementation should all go before testImplementation

Reply:
done